src/dali/abstract_ccxt_input_plugin.py

```diff
@@ -439,6 +439,16 @@ def _process_withdrawals(
             # End of pagination details
             pass
 
+    def _safe_api_call(
+        self,
+        function: Callable[..., Iterable[Dict[str, Union[str, float]]]],
+        params: Dict[str, Any],
+    ) -> Iterable[Dict[str, Union[str, float]]]:
+        return self.__safe_api_call(
+            function,
+            params
+        )
+
     def __safe_api_call(
```

Comment:
You can just make this function protected by prepending only one underscore:
```
def _safe_api_call(
```
Then, we don't need to pass it up. It was private before because it was more specific to the unified API, but should work just fine with the implicit API since we made it generic.

Reply:
Agreed.